.github/workflows/trigger-nightly.yml

```diff
@@ -0,0 +1,46 @@
+name: Trigger nightly build
+
+on:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron:  '0 * * * *'
+  workflow_dispatch:
+
+jobs:
+  trigger-nightly:
+    name: Push tag for nightly build
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: 'Checkout'
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.NIGHTLY_BUILD_GH_TOKEN }}
+      -
+        name: 'Push new tag'
+        # TODO determine nightly version based on latest minor version
+        run: |
+          git config user.name "${GITHUB_ACTOR}"
+          git config user.email "${GITHUB_ACTOR}@users.noreply.github.com"
+
+          TAG=v0.2.0-nightly.$(date +'%s')
+          git tag -a $TAG -m "$TAG: nightly build"
+          git push origin $TAG
```

Comment:
So I guess if we didn't commit anything the tag will be added to the same commit and it is as if nothing happened - we don't trigger another nightly build, right?

Reply:
It will actually trigger another build, since every tag push will trigger a release (regular or nightly).

Reply:
I see, makes sense. Last thing here - would it make sense to change the date format to `YYYYMMDD`? Then the tag would be readable, we would quickly have an idea when the nightly was created.

Reply:
That's a good point. I actually had it like that initially, but changed to the timestamp, because I was running the builds every 15 minutes in my fork.: )

Reply:
Yeah, it should be the last change you do, once you're sure it works 😄

Reply:
Changed date format.